src/Microsoft.Data.Analysis/ColumnArithmeticTemplate.ttinclude

```diff
@@ -432,20 +426,18 @@
                 case MethodType.Unary:
                 case MethodType.UnaryInPlace:
                     throw new ArgumentException();
-                    return $"{dataFrameType}<{genericType}> {Op1Name}";
                 case MethodType.BinaryScalar:
```

Comment:
```csharp
These two can be combined.

                case MethodType.BinaryScalar:
                case MethodType.ComparisonScalar:
                    return $"{genericType} {Op2Name}";
```

Reply:
There are some similar changes that be made for other this above and below. 

Up to you :). Not blocking

Reply:
Fixed